### ISSUE TITLE
error: Error: Command failed:

### ISSUE THREAD

**Issue by fabianmarz** [CodeBuffet/Slekkel#5]
I'm getting this error after providing my credentials. Any suggestions? Username and Password are 100% correct.

Maybe something has changed during the past 7 months?

**Comment by CodyMorris** [comment]
Having same issue

**Comment by fabianmarz** [comment]
Yes, I've also tried to figure out where the mistake is happening, without success...really weird

**Comment by chrisheninger** [comment]
Same issue– solved it by changing the engine from slimerjs to phantomjs.
